apps/chat/src/components/Promptbar/components/PromptModal.tsx

```diff
@@ -166,6 +179,8 @@ export const PromptModal: FC<Props> = ({ isOpen, onClose, onUpdatePrompt }) => {
     setName(selectedPrompt?.name || '');
   }, [selectedPrompt?.name]);
 
+  const saveDisabled = !name.trim() || !content.trim();
```

Comment:
```suggestion
  const saveDisabled = !prepareEntityName(name) || !content.trim();
```

Maybe we should think about forbidden symbols also?